### PR TITLE
OB 2p1 Context (addition of Badge Connect elements)

### PR DESCRIPTION
**Pull request by ahripak**
For issue #228 This PR adds in the elements required for the manifest file introduced by Badge Connect. Those elements become part of the main Open Badge context file.

I wasn't sure of the right place to put this, because the 2p0 folder contains contexts for previous versions as well, but in their own respective "version" folders. Feel free to suggest a different location. I also presume we'll want to remove the generated context file after this is merged.

Big TY to @andyfmiller and @ottonomy for working through this with me last week.

### CODE REVIEW

File: ob_v2p1/schema/jsonld-context/context.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    "Assertion": "obi:Assertion",
+    "BadgeClass": "obi:BadgeClass",
+    "Criteria": "obi:Criteria",
+    "Evidence": "obi:Evidence",
+    "Extension": "obi:Extension",
+    "FrameValidation": "obi:FrameValidation",
+    "IdentityObject": "obi:IdentityObject",
+    "Image": "obi:Image",
+    "HostedBadge": "obi:HostedBadge",
+    "hosted": "obi:HostedBadge",
```
Comment (amiller-ims) [inline ob_v2p1/schema/jsonld-context/context.json]: Should this be "Hosted" with a capital "H"?

File: ob_v2p1/schema/jsonld-context/context.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    "Issuer": "obi:Issuer",
+    "Profile": "obi:Profile",
+    "RevocationList": "obi:RevocationList",
+    "SignedBadge": "obi:SignedBadge",
+    "signed": "obi:SignedBadge",
```
Comment (amiller-ims) [inline ob_v2p1/schema/jsonld-context/context.json]: Should this be "Signed" with a capital "S"?

File: ob_v2p1/schema/jsonld-context/context.json (hunk 1 above)
Comment (ahripak) [reply ob_v2p1/schema/jsonld-context/context.json]: AFAIK, this is a legacy convention from the early days of the v0.5.0 & v1.0 spec (before the introduction of JSON-LD) where the type of verification methods was expressed as `hosted` or `signed`.

I don't advise introducing this breaking change in v2.1 but I would be on board with removing these aliases in v3.

Comment (amiller-ims) [reply ob_v2p1/schema/jsonld-context/context.json]: My mistake. I remembered the aliases, but not the case.

**Pull request merged**